lib/ads/ad_manager.js

```diff
@@ -57,7 +57,7 @@ shaka.ads.AdManager = class extends shaka.util.FakeEventTarget {
    * @export
    */
   initClientSide(adContainer, video) {
-    if (!window['google'] || !google.ima) {
+    if (!window['google'] || !google.ima || !google.ima.AdsRequest) {
       throw new shaka.util.Error(
           shaka.util.Error.Severity.CRITICAL,
```

Comment:
Please change the name of the error (in lib/util/error.js) to CS_IMA_SDK_MISSING as this is what it is now.

Reply:
I changed it!